### ISSUE TITLE
Error in loading data from the DWH in Exercise 2 using Colab

### ISSUE THREAD

**Issue by nati12** [ine-rmotr-curriculum/FreeCodeCamp-Pandas-Real-Life-Example#22]
Anyone also experienced problems with loading data from the DWH using Colab?

The code:

`conn = sqlite3.connect('data/sakila.db')

df = pd.read_sql('''
    SELECT
        rental.rental_id, rental.rental_date, rental.return_date,
        customer.last_name AS customer_lastname,
        store.store_id,
        city.city AS rental_store_city,
        film.title AS film_title, film.rental_duration AS film_rental_duration,
        film.rental_rate AS film_rental_rate, film.replacement_cost AS film_replacement_cost,
        film.rating AS film_rating
    FROM rental
    INNER JOIN customer ON rental.customer_id == customer.customer_id
    INNER JOIN inventory ON rental.inventory_id == inventory.inventory_id
    INNER JOIN store ON inventory.store_id == store.store_id
    INNER JOIN address ON store.address_id == address.address_id
    INNER JOIN city ON address.city_id == city.city_id
    INNER JOIN film ON inventory.film_id == film.film_id
    ;
''', conn, index_col='rental_id', parse_dates=['rental_date', 'return_date'])`

The error message:

---------------------------------------------------------------------------
OperationalError                          Traceback (most recent call last)
[<ipython-input-2-e7bd0fe9801d>](https://6uhl73bom58-496ff2e9c6d22116-0-colab.googleusercontent.com/outputframe.html?vrz=colab-20220421-060046-RC00_443335036#) in <module>()
----> 1 conn = sqlite3.connect('data/sakila.db')
      2 
      3 df = pd.read_sql('''
      4     SELECT
      5         rental.rental_id, rental.rental_date, rental.return_date,

OperationalError: unable to open database file---------------------------------------------------------------------------

**Comment by Alekhyadaspet** [comment]
Yes, same error, while connecting from google colab. I tried with mysqlworkbench desktop app, it throws errors in retrieving specific tables; tried by connecting to drive by uploading db installed on local computer, the codeset presented doesnt work for retrieving tables from db as specified in the course; Also tried few things in connecting sqlite3 api from gcp, I was stuck wondering if it is going to cost. 
Tried this code for connecting to drive and mount db to use:
<img width="502" alt="image" src="https://user-images.githubusercontent.com/9607118/184928382-51871c7f-b90e-492e-8fab-22dbff9b1cd7.png">

Error: OperationalError: (sqlite3.OperationalError) unable to open database file
(Background on this error at: https://sqlalche.me/e/14/e3q8)

Error details: 
OperationalError                          Traceback (most recent call last)
[/usr/local/lib/python3.7/dist-packages/sqlalchemy/engine/base.py](https://localhost:8080/#) in _wrap_pool_connect(self, fn, connection)
   3360         try:
-> 3361             return fn()
   3362         except dialect.dbapi.Error as e:

<img width="415" alt="image" src="https://user-images.githubusercontent.com/9607118/184927247-a33ceb32-cde6-476c-9859-ef1cce3da84f.png">

<img width="415" alt="image" src="https://user-images.githubusercontent.com/9607118/184927430-c6c25d16-8711-4315-97ff-e5620a923cec.png">

**Comment by santiagobasulto** [comment]
This is probably related to Google's management of the file system in Colab. I'm leaving this open so people can brainstorm solutions.
